### PR TITLE
test: rbac config [TESTENG-108]

### PR DESCRIPTION
**Pull request by JComins000**
## Ticket

[TESTENG-108]

## Description

adds rbac devcluster config and uses it in e2e-react-ee

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[TESTENG-108]: https://hpe-aiatscale.atlassian.net/browse/TESTENG-108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/devcluster/react-rbac.devcluster.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      name: db
+
+  - master:
+      pre:
+        - sh: make -C tools prep-root
+      config_file:
+        security:
+          initial_user_password: $INITIAL_USER_PASSWORD
+          authz:
+            rbac_ui_enabled: true
```
Comment (JComins000) [inline .circleci/devcluster/react-rbac.devcluster.yaml:14]: 👁️ 👁️

**Comment by johnkim-det** [comment]
looks like there are quite a few tests that fail with `rbacEnabled: true`, even though we don't have any tests explicitly requiring RBAC yet -- maybe the admin user needs to have `ClusterAdmin` role added when `rbacEnbaled: true`?

**Review by amandavialva01 — APPROVED** [review]
backend changes LGTM!

**Review by davidfluck-hpe — APPROVED** [review]

**Comment by johnkim-det** [comment]
@JComins000 went ahead and added fixes for test failures. Hopefully we can move forward with the PR if that looks good.

**Review by djanicekpach — APPROVED** [review]

**Pull request merged**